### ISSUE TITLE
Question about dataset

### ISSUE THREAD

**Issue by Baymine** [ExplainableML/ACVC#1]
Thank you for sharing your marvelous work! 

I just want to know what the difference between `DomainNet` and `FullDomainNet`.  Can I change `FullDomainNet` into `DomainNet`? And how?

Thank you again!

**Comment by cuguilke** [comment]
When you select `DomainNet`, the data loader just picks the 10 common classes with COCO dataset. So, the use case is `--train_dataset COCO --test_datasets DomainNet:Real DomainNet:DomainNet:Infograph ...`. This approach corresponds to the [case study](https://github.com/ExplainableML/ACVC#case-study-coco-benchmark) explained in the readme file.

`FullDomainNet:Real`, on the other hand, trains with all 345 classes. This case also requires quite a lot of memory, so , if you check the codes, we actually use a small validation set defined in `labelfile = os.path.join(data_path, "fast.txt")` during training and set `--save_model` while running the training script. After the training is done, we load each test domain and obtain the performance results using the saved model one by one since we were not able to fit multiple domains into the memory at the same time.
